src/ActionList/Item.tsx

```diff
@@ -0,0 +1,88 @@
+import type {IconProps} from '@primer/octicons-react'
+import React from 'react'
+import styled from 'styled-components'
+import {get} from '../constants'
+
+interface ItemPropsBase extends React.ComponentPropsWithoutRef<'div'> {
+  text: string
+  description?: string
+  descriptionVariant?: 'inline' | 'block'
+  leadingVisual?: React.FunctionComponent<IconProps>
+  leadingVisualSize?: 16 | 20
+  size?: 'small' | 'medium' | 'large'
+  variant?: 'default' | 'singleSelection' | 'multiSelection' | 'danger' | 'static'
+}
+interface ItemPropsWithRenderItem extends Partial<ItemPropsBase> {
+  renderItem: (props: ItemProps) => JSX.Element
+}
+export type ItemProps = ItemPropsBase | ItemPropsWithRenderItem
+
+const StyledItem = styled.div<{variant: ItemProps['variant']}>`
+    position: relative;
+    display: flex;
+    align-items: start;
+    border-radius: ${get('radii.2')};
+    font-weight: normal;
+    color: ${({variant}) => (variant === 'danger' ? get('colors.text.danger') : 'inherit')};
+    text-decoration: none;
+    border: 0;
```

Comment:
Why is `border` set to `0`? Isn't that the default? Why is there a `border-radius` set above of the border is 0?

Reply:
Removed in https://github.com/primer/components/pull/1078/commits/90fea7a1ef3631bf407106bdb308145c1d6998f5!